klt-cert-manager/go.mod

```diff
@@ -8,66 +8,66 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/afero v1.9.5
 	github.com/stretchr/testify v1.8.4
-	k8s.io/api v0.26.9
-	k8s.io/apiextensions-apiserver v0.26.9
-	k8s.io/apimachinery v0.26.9
-	k8s.io/client-go v0.26.9
-	sigs.k8s.io/controller-runtime v0.14.6
+	k8s.io/api v0.28.1
```

Comment:
can we already update all of those to 0.28.2? seems like that version has just been released last week

Reply:
bumped